python/ray/_private/utils.py

```diff
@@ -78,6 +78,14 @@
 
 
 def get_user_temp_dir():
+    # When global mode is enabled in databricks runtime, ray.init() looks for
+    # existing ray cluster in corresponding default temp dir.
+    # This should be consistent with python/ray/util/spark/databricks_hook.py.
+    if (
+        os.environ.get("DATABRICKS_RAY_CLUSTER_GLOBAL_MODE", "false").lower() == "true"
+        and "DATABRICKS_RUNTIME_VERSION" in os.environ
+    ):
+        return "/local_disk0/tmp"
     if "RAY_TMPDIR" in os.environ:
```

Comment:
@serena-ruan 

What about we don't change https://github.com/ray-project/ray/pull/39313/files#r1317953317, but add a default `RAY_TMPDIR` environment pointing to "/local_disk0/tmp"  in databricks runtime ?